docs/reference/requirement-sets/shared-runtime-requirement-sets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Shared runtime requirement sets
+description: 'Specifies the platforms and Office hosts that support the SharedRuntime APIs.'
+ms.date: 01/21/2020
+ms.prod: non-product-specific
+localization_priority: Normal
+---
+
+# Shared runtime requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
```

Comment:
Suggestion for improved maintainability -- 

Rather than duplicating this same paragraph in each of the 6 'common' requirement set articles, you might consider moving this content to an include file and then updating each of the requirement set articles to use the include file in place of this paragraph.

Reply:
Good idea. But I'll put that in a separate PR.